consumers/email-consumer/Dockerfile

```
@@ -0,0 +1,16 @@
+FROM python:3.13.2-slim-bookworm
+LABEL maintainer="segfault"
+
+ENV PYTHONUNBUFFERED=1
+
+COPY ./requirements.txt /tmp/requirements.txt
+EXPOSE 8000
+
+ARG DEV=false
+RUN pip install --upgrade pip && \
```

Hadolint (2.12.0)

[warning] Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>` or `pip install --requirement <requirements file>`

(DL3013)